tinnsleep/scoring.py

```diff
@@ -70,15 +70,17 @@ def rearrange_chronological(brux_list):
     return brux_list
 
 
-def burst_to_episode(burst_list, delim=3):
+def burst_to_episode(burst_list, delim=3, min_burst_joining=2):
     """ Transforms a chronological list of bursts into a 
     chronological list of episodes
     
     Parameters
     ----------
     burst_list : list of burst instances
-    delim: float,
+    delim: Optional, float,
```

Comment:
you don't need to specify `optional` but to say:
`     delim : float (default: 2.0) `